systers_portal/users/models.py

```diff
@@ -21,6 +23,9 @@ class SystersUser(models.Model):
                                         blank=True,
                                         null=True,
                                         verbose_name="Profile picture")
+    profile_picture_thumbnail = ImageSpecField(source='profile_picture',
+                                               processors=[ResizeToFill(1000, 1000)],
```

Comment:
Done, I changed it 1000 because in chrome favicon icon image was not looking at all, but as you said it could be because of cache.